src/js/controllers/create.js

```diff
@@ -27,7 +27,7 @@ angular.module('copayApp.controllers').controller('createController',
       var defaults = configService.getDefaults();
       var tc = $state.current.name == 'tabs.add.create-personal' ? 1 : defaults.wallet.totalCopayers;
```

Comment:
Can you rename tc to something less cryptic? totalCopayers perhaps?

Reply:
You might also want to think about breaking out the BCH URL as a separate variable to express your intent and to avoid doing too much on one line.

Reply:
Implemented suggestions.